packages/web/src/pages/audio-rewards-page/Tiers.module.css

```diff
@@ -5,7 +5,7 @@
   background-color: var(--white);
   flex: 1;
   justify-content: flex-start;
-  border-radius: 6px;
+  border-radius: 12px;
```

Comment:
What are these for? Are they supposed to be part of this commit?

Reply:
yeah these border radiuses have been wrong for a while - confirmed with Julian/Forrest we want these to be 12px!